.genx/prompts/ui.js

```diff
@@ -10,13 +10,22 @@ module.exports = async (inquirer, prevAns = {}) => {
     {
       name: 'routes',
       type: 'input',
-      message: 'Pages in comma separated format e.g. home,dashboard,admin',
+      message: 'Pages config in json format',
       when: !prevAns.routes,
-      default: prevAns.routes || 'home',
+      default: `[{name: 'home'}]`,
     },
   ])
 
+  let routesParsed = [{name: 'home'}];
+  if (routes) {
+    try {
+      routesParsed = JSON.parse(routes);
+    } catch (error) {
+      console.error("Error parsing JSON, will use default routes config:", error);
```

Comment:
Not passing `routes` results in below, which looks like there's something wrong:

```
Error parsing JSON, will use default routes config: SyntaxError: Expected property name or '}' in JSON at position 2
    at JSON.parse (<anonymous>)
    at module.exports (/Users/artur.cistov/code/prtest2/.genx/prompts/ui.js:22:27)
    at async module.exports (/Users/artur.cistov/code/prtest2/.genx/prompts.js:16:20)
```

It should be like this: you didn't specify routes, we silently we switch to the default.
If you did specify, but it's invalid JSON, we should print just `error.message` (without the stacktrace) - something like this:

```
console.error("Error parsing `routes` parameter as JSON:", error.message);
console.log("Falling back to the default routes value");
```